openlibrary/plugins/openlibrary/js/ile/utils/SelectionManager/SelectionManager.js

```diff
@@ -23,8 +23,10 @@ export default class SelectionManager {
         this.curpath = curpath;
         this.inited = false;
         this.selectedItems = {};
+        this.lastSelectedIndex = -1;
```

Comment:
Done. Because of the sibling issue discussed above, I've also added a function to generate a list of "siblings" that need to be worked on, given the last change and the current target.